jina/clients/python/grpc.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, args: 'argparse.Namespace'):
         self.logger.debug('setting up grpc insecure channel...')
         # A gRPC channel provides a connection to a remote gRPC server.
         self._channel = grpc.insecure_channel(
-            '%s:%d' % (args.host, args.port_expose),
+            f'{args.host}:{args.port_expose:d}',
```

Comment:
this `:d` seems odd

Reply:
It's a formatting code, like the 's' and 'f'. But yeah, it doesn't add anything. I can remove them